examples/kerberos.rs

```diff
@@ -0,0 +1,150 @@
+use base64::Engine;
+use hyper::header::{
+    ACCEPT, ACCEPT_ENCODING, ACCEPT_LANGUAGE, AUTHORIZATION, CONNECTION, CONTENT_LENGTH, HOST, USER_AGENT,
+};
+use hyper::StatusCode;
+use sspi::builders::EmptyInitializeSecurityContext;
+use sspi::{
+    AcquireCredentialsHandleResult, ClientRequestFlags, CredentialsBuffers, DataRepresentation,
+    InitializeSecurityContextResult, KerberosConfig, SecurityBuffer, SecurityBufferType, SecurityStatus, Sspi,
+};
+use sspi::{Kerberos, SspiImpl};
+use std::error::Error;
+
+static HOSTNAME: &'static str = "computer_name.domain";
+static USERNAME: &'static str = "user@domain";
+static PASSWORD: &'static str = "Passoword";
+static AUTH_METHOD: &'static str = "Negotiate"; // Negotiate or Kerberos
+
+fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
+    let kdc_url = std::env::var("SSPI_KDC_URL").expect("a KDC URL set in SSPI_KDC_URL");
```

Comment:
use env var for kdc url, not sure if we should use env var for other variables as well. Please let me know and I can update accordingly. Thanks.

Reply:
Yes please, it should not be required to modify the source code in order to run the example.
https://github.com/Devolutions/sspi-rs/pull/178#discussion_r1349321226
https://github.com/Devolutions/sspi-rs/pull/178#discussion_r1349321721